### PR TITLE
Chore/package update json schema viewer

### PR DESCRIPTION
**Pull request by brendarearden**
# Elements Default PR Template

In general, make sure you have: (check the boxes to acknowledge you've followed this template)

- [x] Read [`CONTRIBUTING.md`](../CONTRIBUTING.md)

#### Other Available PR Templates:

- Release: https://github.com/stoplightio/elements/compare?template=release.md
  - [x] [Read the release section of `CONTRIBUTING.md`](../CONTRIBUTING.md#releasing-elements)

## Description
https://github.com/stoplightio/json-schema-viewer/pull/198 changes to json-schema-viewer prompted package version bump in elements-core.

### CODE REVIEW

File: packages/elements-dev-portal/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-dev-portal",
-  "version": "1.7.3",
+  "version": "1.7.4",
   "description": "UI components for composing beautiful developer documentation.",
   "keywords": [],
   "sideEffects": [
```
Comment (kaylachun) [inline packages/elements-dev-portal/package.json:6]: Line 62: does elements-core version need bumping here?

Comment (brendarearden) [reply packages/elements-dev-portal/package.json:6]: good catch - I have bumped the version on line 62!

File: packages/elements/package.json [hunk 2]
```diff
@@ -58,9 +58,9 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@stoplight/elements-core": "~7.5.19",
```
[the remark this answers is not in the document]
Comment (brendarearden) [reply packages/elements/package.json]: This has been bumped in the most recent commit

**Review by kaylachun — APPROVED** [review]
Looks good!

**Review by kaylachun — APPROVED** [review]

**Pull request merged**